src/cascadia/TerminalSettingsModel/ActionMap.cpp

```diff
@@ -964,17 +965,32 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
             cmdImpl.copy_from(winrt::get_self<implementation::Command>(command));
 
             const auto inArgs{ command.ActionAndArgs().Args().try_as<Model::SendInputArgs>() };
-
+            const auto inputString{ (std::wstring_view)(inArgs ? inArgs.Input() : L"") };

```

Comment:
Technically you need to assign the `Input()` to somewhere before getting a weak string-view to it. It's not guaranteed that the implementation of the property holds onto a reference of the `HSTRING` after all.

Reply:
I'm gonna block over this, since it could be an easy safety issue